parser.c

```diff
@@ -149,6 +166,10 @@ static bool parser_next_token(struct kanshi_parser *parser) {
 			parser->tok_type = KANSHI_TOKEN_STR;
 			parser->tok_str_len = 0;
 			return parser_read_quoted(parser);
+		} else if (ch == commentSign) {
+            parser->tok_type = KANSHI_TOKEN_COMMENT;
```

Comment:
I think we can just consume the comment here and throw it away. No need to have `KANSHI_TOKEN_COMMENT` (and to handle it in the caller).

Reply:
Do you have a .clang-format for your project I can use to make sure that code is formatted the way you like it?

Reply:
Unfortunately no. clang-format is missing a feature necessary for applying the sway/wlroots style.

Reply:
about line: 169; this function `parser_next_token` is actually used in parse_profile_output and the whole line is discarded there. This is needed for comments: 
```
{
output ... # comment is next to output
```